### PR TITLE
[build][test][mxnet|tensorflow|pytorch] Update Smdebug Binary

### PR DESCRIPTION
**Pull request by NihalHarish**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

#### EIA Checklist
* When creating a PR:
- [ ] I've modified ```src/config/build_config.py``` in my PR branch by setting ```ENABLE_EI_MODE = True```
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/training/docker/1.15.3/py36/cu100/Dockerfile.gpu [hunk 1]
```diff
@@ -24,7 +24,7 @@ ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.15.3/AmazonLinux/
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```
Comment (saimidu) [inline tensorflow/training/docker/1.15.3/py36/cu100/Dockerfile.gpu:27]: Add this to 1.15.3 py3 as well, which contains the dockerfiles for Python 3.7

File: tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu [hunk 2]
```diff
@@ -27,7 +27,7 @@ ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.15.3/AmazonLinux/
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```
Comment (saimidu) [inline tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu:30]: Add this to 1.15.3 py3 as well, which contains the dockerfiles for Python 3.7

File: tensorflow/training/docker/2.0.1/py3/Dockerfile.cpu [hunk 3]
```diff
@@ -28,7 +28,7 @@ ARG PIP=pip3
 ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.1/AmazonLinux/cpu/final/tensorflow-2.0.1-cp36-cp36m-manylinux2010_x86_64.whl
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```
Comment (saimidu) [inline tensorflow/training/docker/2.0.1/py3/Dockerfile.cpu:31]: Add this to 2.0.2 py3 as well

File: tensorflow/training/docker/2.0.1/py3/cu100/Dockerfile.gpu [hunk 4]
```diff
@@ -28,7 +28,7 @@ ARG PIP=pip3
 ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.1/AmazonLinux/gpu/final/tensorflow_gpu-2.0.1-cp36-cp36m-manylinux2010_x86_64.whl
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```
Comment (saimidu) [inline tensorflow/training/docker/2.0.1/py3/cu100/Dockerfile.gpu:31]: Add this to 2.0.2 py3 as well

File: tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu [hunk 5]
```diff
@@ -28,13 +28,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=s3://aws-tensorflow-binaries/tensorflow/r2.2_aws/20200806-193742/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-linux_x86_64.whl
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu:31]: Can't pip install from an S3 URI. Please use an HTTPS link here.

File: tensorflow/training/docker/2.2.0/py3/cu102/Dockerfile.gpu [hunk 6]
```diff
@@ -27,13 +27,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/gpu/py37/tensorflow_gpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=s3://aws-tensorflow-binaries/tensorflow/r2.2_aws/20200806-193742/gpu/py37/tensorflow_gpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/cu102/Dockerfile.gpu:30]: ARG TF_URL=s3://aws-tensorflow-binaries/tensorflow/r2.2_aws/20200806-193742/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-linux_x86_64.whl

File: tensorflow/training/docker/1.15.3/py36/cu100/Dockerfile.gpu (hunk 1 above)
Comment (NihalHarish) [reply tensorflow/training/docker/1.15.3/py36/cu100/Dockerfile.gpu:27]: Done.

File: tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu (hunk 2 above)
Comment (NihalHarish) [reply tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu:30]: Done.

File: tensorflow/training/docker/2.0.1/py3/Dockerfile.cpu (hunk 3 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.0.1/py3/Dockerfile.cpu:31]: Done.

File: tensorflow/training/docker/2.0.1/py3/cu100/Dockerfile.gpu (hunk 4 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.0.1/py3/cu100/Dockerfile.gpu:31]: Done.

File: tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu (hunk 5 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu]: Done.

File: tensorflow/training/docker/1.15.3/py3/Dockerfile.cpu [hunk 7]
```diff
@@ -135,7 +135,7 @@ RUN ${PIP} install --no-cache-dir -U \
     Pillow \
     h5py==2.10.0 \
     requests==2.22.0 \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```
Comment (saimidu) [inline tensorflow/training/docker/1.15.3/py3/Dockerfile.cpu:138]: Please make this an ARG like the other dockerfiles

File: tensorflow/training/docker/1.15.3/py3/cu100/Dockerfile.gpu [hunk 8]
```diff
@@ -176,7 +176,7 @@ RUN ${PIP} install --no-cache-dir -U \
     Pillow \
     h5py==2.10.0 \
     requests==2.22.0 \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```
Comment (saimidu) [inline tensorflow/training/docker/1.15.3/py3/cu100/Dockerfile.gpu:179]: Please make this an ARG like the other dockerfiles

File: tensorflow/training/docker/2.0.2/py3/Dockerfile.cpu [hunk 9]
```diff
@@ -133,7 +133,7 @@ RUN ${PIP} install --no-cache-dir -U \
     "sagemaker<2" \
     sagemaker-experiments==0.* \
     "sagemaker-tensorflow>=2.0,<2.1" \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```
Comment (saimidu) [inline tensorflow/training/docker/2.0.2/py3/Dockerfile.cpu:136]: Please make this an ARG like the other dockerfiles

File: tensorflow/training/docker/2.0.2/py3/cu100/Dockerfile.gpu [hunk 10]
```diff
@@ -176,7 +176,7 @@ RUN ${PIP} install --no-cache-dir -U \
     sagemaker-experiments==0.* \
     "sagemaker-tensorflow>=2.0,<2.1" \
     "sagemaker-tensorflow-training>=3.0<4.0" \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```
Comment (saimidu) [inline tensorflow/training/docker/2.0.2/py3/cu100/Dockerfile.gpu:179]: Please make this an ARG like the other dockerfiles

File: tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu [hunk 11]
```diff
@@ -28,13 +28,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200810-220210/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-linux_x86_64.whl
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu:31]: This needs to be "manylinux2010_x86_64", right?

File: tensorflow/training/docker/1.15.3/py3/Dockerfile.cpu (hunk 7 above)
Comment (NihalHarish) [reply tensorflow/training/docker/1.15.3/py3/Dockerfile.cpu]: Done

File: tensorflow/training/docker/1.15.3/py3/cu100/Dockerfile.gpu (hunk 8 above)
Comment (NihalHarish) [reply tensorflow/training/docker/1.15.3/py3/cu100/Dockerfile.gpu]: Done.

File: tensorflow/training/docker/2.0.2/py3/Dockerfile.cpu (hunk 9 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.0.2/py3/Dockerfile.cpu]: Done

File: tensorflow/training/docker/2.0.2/py3/cu100/Dockerfile.gpu (hunk 10 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.0.2/py3/cu100/Dockerfile.gpu]: Done.

File: tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu (hunk 11 above)
Comment (NihalHarish) [reply tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu]: Corrected this.

File: tensorflow/training/docker/2.2.0/py3/cu101/Dockerfile.gpu [hunk 12]
```diff
@@ -33,7 +33,7 @@ ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/est
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/cu101/Dockerfile.gpu:36]: Does the TF_URL on this dockerfile not need to be updated like the other dockerfiles?

File: tensorflow/training/docker/2.2.0/py3/cu102/Dockerfile.gpu [hunk 13]
```diff
@@ -27,13 +27,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/gpu/py37/tensorflow_gpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200810-220210/gpu/py37/tensorflow_gpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
 
 ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/estimator/r2.2_aws/20200801-180340/tensorflow_estimator-2.2.0-py2.py3-none-any.whl
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/cu102/Dockerfile.gpu:32]: Does ESTIMATOR_URL not need to be modified alongside TF_URL?

File: tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu [hunk 14]
```diff
@@ -28,13 +28,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200810-220210/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
 
 ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/estimator/r2.2_aws/20200801-180340/tensorflow_estimator-2.2.0-py2.py3-none-any.whl
```
Comment (saimidu) [inline tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu:33]: Does ESTIMATOR_URL not need to be modified alongside TF_URL?

Comment (NihalHarish) [reply tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu:33]: There are no changes to the estimator binary that are necessary for the functioning of smdebug

**Pull request merged**